src/Symfony/Component/Security/Core/Resources/translations/security.pt_BR.xlf

```diff
@@ -64,7 +64,15 @@
             </trans-unit>
             <trans-unit id="16">
                 <source>Account is locked.</source>
-                <target>A conta está travada.</target>
+                <target>A conta está bloqueada.</target>
```

Comment:
I think using "travada" is better here. 

It should be "locked", and not "blocked". 
Ie, you can "lock" an issue on Github without "blocking" anyone.

Reply:
Thanks for the suggestion. Fixed.